src/backends/swift/objects.ts

```
@@ -1248,54 +1234,78 @@ export async function listMultipartUploads(
 
-  // Parse the index file
-  let uploads = [];
+  // Parse the list and filter for .json files
+  let uploadJsonFiles: string[] = [];
   try {
-    const indexData = await unwrapOk(indexResponse).json();
-    uploads = Array.isArray(indexData.uploads) ? indexData.uploads : [];
+    const listData = await unwrapOk(listResponse).json();
+    uploadJsonFiles = (listData || [])
+      .filter((item: { name: string }) =>
+        item.name && item.name.startsWith(multipartIndexPrefix) &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)